src/state/swap/hooks.ts

```diff
@@ -220,22 +224,29 @@ export function useDerivedSwapInfo(platformOverride?: RoutablePlatform): UseDeri
       },
     }
 
+    // Use a static version
+    const staticProvider = provider ? new StaticJsonRpcProvider(provider?.connection.url) : undefined
+
+    console.log('useDerivedSwapInfo: fetching trades')
```

Comment:
Do we want to log this?

Reply:
We have many logs :P from other places. I will keep it there until next version of EcoRouter :)